### DISCUSSION TITLE
Add hotkey for unsubscribing from notifications

### DISCUSSION THREAD

**Discussion by peppy** [community/community#65] [Mobile]
My two most common actions with notifications are to mark as done (in the case the issue still relevant but needs a response from someone else) or unsubscribe (i don't need to worry about this particular issue).

On the web, Shift-M handily allows unsubscribing from selected notifications. On mobile there doesn't seem to be an equivalent (doesn't show in hold-cmd hotkeys at least) and the only way seems to be using swipe. Would be nice to have this!

**Comment by candyho** [comment]
Hey thanks for sharing your feedback here again! Cross-referencing [related post](https://github.com/github/feedback/discussions/42) here for our engineers to learn more context.  Keyboard shortcuts on iPad is something on our backlog. Will keep you posted once we land on this.
